test/datasets_utils.py

```diff
@@ -166,9 +166,11 @@ class DatasetTestCase(unittest.TestCase):
 
     Optionally, you can overwrite the following class attributes:
 
-        - CONFIGS (Sequence[Dict[str, Any]]): Additional configs that should be tested. Each dictonary can contain an
+        - CONFIGS (Sequence[Dict[str, Any]]): Additional configs that should be tested. Each dictionary can contain an
```

Comment:
Should we remove "additional" ? It's not clear what other configs are tested apart from the ones in `CONFIGS`. It seems that the `DEFAULT_CONFIG` isn't tested by `test_all_configs`